src/bin/flamegraph.rs

```diff
@@ -131,6 +133,18 @@ struct Opt {
     )]
     fontwidth: f64,
 
+    /// Color of UI text such as the search and reset zoom buttons
+    #[clap(
+        long = "uicolor",
+        default_value = defaults::UI_COLOR,
+        value_parser = |s: &str| {
```

Comment:
`value_parser` was added in [`clap 3.2.0`](https://github.com/clap-rs/clap/blob/master/CHANGELOG.md#320---2022-06-13), so you'll have to update `Cargo.toml` to have that be the required version, then this should work :+1: